### DISCUSSION TITLE
limit metrics runs to specific repositories

### DISCUSSION THREAD

**Discussion by dkhokhlov** [lowlighter/metrics#1391] [Q&A]
Is it possible to limit metrics runs to specific repositories?
limit to specific repositories in organizations in order to reduce GH api calls.

**Comment by lowlighter** [comment]
Hi 👋 !

It's kind of supported by using the [Advanced pattern matching of `repositories_skipped` option](https://github.com/lowlighter/metrics/blob/master/source/plugins/core/README.md#-skipping-repositories-in-plugins):

```yaml
-org/*
+org/include-this-repo
```

It will prevent plugins that support skipping repositories from fetching additional data of excluded repositories, but some plugins that uses "global stats" (like total count) may still report them

Eventually when GitHub scoped repositories are supported, it'll be easier to indicate which repositories should be included I guess
